AzureDevOpsPowerShell/Public/Api/Core/TeamSettings/Get-AzDoTeamSettings.ps1

```diff
@@ -0,0 +1,83 @@
+function Get-AzDoTeamSettings {
+  <#
+.SYNOPSIS
+    Get the settings of a Team in Azure DevOps.
+.DESCRIPTION
+    Get the settings of a Team or multiple in Azure DevOps.
+.EXAMPLE
+    $Params = @{
+      CollectionUri  = "https://dev.azure.com/cantoso"
+      ProjectName    = "Playground"
+      TeamName           = "Team1"
+    }
+
+    Get-AzDoTeamSettings @Params
+
+    This example will get the settings of the team 'Team1' in the project 'Playground'.
+.OUTPUTS
+    PSObject with the settings of the team(s).
+#>
+
+  [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Low')]
+  param (
+    # Collection Uri of the organization
+    [Parameter(Mandatory, ValueFromPipelineByPropertyName)]
+    [ValidateScript({ Validate-CollectionUri -CollectionUri $_ })]
+    [string]
+    $CollectionUri,
+
+    # Name of the project where the team is located
+    [Parameter(Mandatory, ValueFromPipelineByPropertyName)]
+    [string]
+    $ProjectName,
+
+    # Name of the team to get the settings from
+    [Parameter(Mandatory, ValueFromPipelineByPropertyName)]
+    [string[]]
+    $TeamName
+  )
+
+  begin {
+    Write-Verbose "Starting function: Get-AzDoTeamSettings"
+    $CollectionUri = $CollectionUri.TrimEnd('/')
+    # create dynamic array to store the results
+    $result = New-Object System.Collections.Generic.List[System.Object]
+  }
+
+  process {
+    foreach ($Name in $TeamName) {
+      $params = @{
```

Comment:
We should implement getting a team before trying to get it's settings.
Getting settings from a team that doesn't exist results in:
`OperationStopped: The team with id 'test5' does not exist, or you do not have permission to access it.`
Should wait on #68